.github/workflows/docker-publish.yml

```
@@ -0,0 +1,35 @@
+name: Semantic Release
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  Release:
```

Suggest using the `name` key to explicitly assign a name to the job instead.
```suggestion
  release:
    name: Release
```